interface/wx/event.h

```diff
@@ -3892,7 +3892,121 @@ class wxScrollEvent : public wxCommandEvent
     void SetPosition(int pos);
 };
 
+/** @class wxTouchEventBase
+    This is the base class for all touch events.
 
+    @note Touch events are not generated by default, you must call
+          wxWindow::EnableTouchEvents() with the appropriate parameter to
+          request their generation.
+
+    @library{wxcore}
+    @category{events}
+
+    @see wxGestureEvent, wxMultiTouchEvent
+
+    @since 3.3.0
+*/
+class wxTouchEventBase : public wxEvent
+{
+    /**
+        Constructor.
+    */
+    wxTouchEventBase(wxWindowID winid = 0, wxEventType type = wxEVT_NULL);
+
+    /**
+        Returns the position where the event took effect, in client coordinates: position of Pan event,
```

Comment:
We should have "brief" description, appearing in the summary in the docs, and for this we need a fullstop here, e.g.
```suggestion
        Returns the position where the event took effect, in client coordinates.
        
         This is the position of Pan event,
```